lib/src/component_declaration/component_base_2.dart

```diff
@@ -521,7 +509,6 @@ abstract class UiComponent2<TProps extends UiProps> extends react.Component2
 ///
 /// __Initializing state:__
 ///
-///     @Component2()
 ///     class YourComponent extends UiStatefulComponent2<YourProps, YourState> {
 ///       @override
 ///       void init() {
```

Comment:
Woops... I know this is out-of-scope for this PR... but lets clean this up while we're here.

Reply:
💯